src/components/ProjectList/ProjectListContainer/index.jsx

```diff
@@ -1,18 +1,44 @@
 import React from 'react';
 
-import { rtd } from './projectListContainer.module.scss';
+import { rtc } from './projectListContainer.module.scss';
 
-const ProjectListContainer = ({ children }) => (
-  <table>
-    <thead>
-      <tr>
-        <th>Project Health</th>
-        <th>Project Name</th>
-        <th className={rtd}>Last Updated</th>
-      </tr>
-    </thead>
-    <tbody>{children}</tbody>
-  </table>
-);
+const ProjectListContainer = ({ children, status }) => {
+  const statusTitleArr = [];
+  var i;
```

Comment:
Curious, was there a reason you used `var` here?

Reply:
Nope. Just a habit from when I learned how to use for loops